_data/simple-icons.json

```diff
@@ -2045,6 +2045,11 @@
             "hex": "FAAD4D",
             "source": "http://www.wix.com/about/design-assets"
         },
+        {
+            "title": "Wolfram",
```

Comment:
Although the technically accurate name is `"Wolfram Research"`, it is never used except where distinguishing between Wolfram and Wolfram products is important. As such I feel that `"Wolfram"` - which is used practically everywhere that the company refers to itself - is acceptable.